CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 find_package(Threads REQUIRED)
+find_package(nlohmann_json 3.2.0 REQUIRED)
```

Comment:
Find JSON should be in src/core/CMakeLists.txt, beacouse only core will use it for now